.github/workflows/test.yml

```diff
@@ -42,7 +44,7 @@ jobs:
           version: "1.4"
 
       - name: Cargo binstall cargo-risczero
-        run: cargo binstall -y --force cargo-risczero@0.19.1
+        run: cargo binstall -y --force cargo-risczero@0.20.0-rc.2
```

Comment:
Currently bonsai staging is at rc3, so might be worth bumping up to that for testing.